Data Structures/Linked-Lists/All implementations of linked list.txt

```diff
@@ -0,0 +1,332 @@
+/*  Program of single linked list*/
+
+
+#include<stdio.h>
+#include<stdlib.h>
+
+struct node
+{
+	int info;
+	struct node *link;
+};
+
+struct node *create_list(struct node *start);
+void display(struct node *start);
+void count(struct node *start);
+void search(struct node *start,int data);
+struct node *addatbeg(struct node *start,int data);

```

Comment:
Follow [Coding Standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards). Use loweCamelCase for function name. Fix this in all files